src/libs/actions/IOU.js

```diff
@@ -957,7 +966,7 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
         optimisticReportActionsData.value[optimisticCreatedAction.reportActionID] = optimisticCreatedAction;
 
         // If we're going to fail to create the report itself, let's not have redundant error messages for the IOU
-        failureData[0].value[optimisticIOUReportAction.reportActionID] = {pendingAction: null};
+        failureData[0].value[optimisticIOUReportAction.reportActionID] = {errors: ErrorUtils.getMicroSecondOnyxError(null)};
```

Comment:
Same not as above. Do not overwrite but set the errors field correctly at the first initialisation. 
Also we are no longer clearing the pendingAction is this change required?

Reply:
Clearing the `pendingAction` is not required if there's an error, this is done only when the API write has succeeded. This was done before just to remove redundant errors.